### ISSUE TITLE
Copilot feature for accept word/line

### ISSUE THREAD

**Issue by Cthutu** [AstroNvim/astrocommunity#149]
Copilot has a feature for accepting the next word and the next line of the suggestion.  Could these be mapped to keybindings (e.g. Ctrl-L/Ctrl-Right for accept word and Ctrl-J/Ctrl-Down for accept line) in the completion.copilot-cmp pack?

**Comment by Cthutu** [comment]
Added PR for this: https://github.com/AstroNvim/astrocommunity/pull/150

**Comment by namefailed** [comment]
You can exit the default configuration like this! these are my bindings for accepting words etc.

```
  { import = "astrocommunity.completion.copilot-lua" },
  {
    "copilot.lua",
    opts = {
      panel = {
        auto_refresh = true,
        keymap = {
          accept = "<C-l>",
          jump_next = "<C-j>",
          jump_prev = "<C-k>",
          dismiss = "<C-h>",
        },
        layout = {
          position = "right",
        },
      },
      suggestion = {
        keymap = {
          accept = "<C-l>",
          accept_word = false,
          accept_line = false,
          next = "<C-j>",
          prev = "<C-k>",
          dismiss = "<C-h>",
        },
      },
    },
  },

```

**Comment by mehalter** [comment]
Closed with #150
